resources/test/tests/functional/idlharness/IdlInterface/test_exposed_star.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE HTML>
```

Comment:
Small nit: I'd call this `test_exposed_wildcard.html` to match the terminology in Web IDL.